### ISSUE TITLE
Add Priestley Timeline to vis-common-plots.html#priestly-timeline

### ISSUE THREAD

**Issue by timsteeno** [aeturrell/coding-for-economists#77]
Noticed that the Priestley timeline at /vis-common-plots.html#priestly-timeline was incomplete. This code or some variation of it should complete it - I can make a pull request.

```
df = df.sort_values('Born')

# Create the plot
fig, ax = plt.subplots(figsize=(12, 6))

for i, (index, row) in enumerate(df.iterrows()):
    lifespan = (row['Died'] - row['Born']).days
    bar = ax.barh(len(df) - 1 - i, lifespan, left=row['Born'], height=0.5)
    text_x = row['Born'] + pd.Timedelta(days=lifespan/2)
    
    # Add text inside the bar
    ax.text(text_x, len(df) - 1 - i, row['Name'], 
            va='center', ha='center', 
            color='white', fontweight='bold', fontsize=8)

ax.set_yticks([])
plt.xlabel('Year')
plt.show()
```

![image](https://github.com/user-attachments/assets/e0da3a1f-f2f3-436d-8bd0-eab96137d266)

**Comment by aeturrell** [comment]
Thank you. Would welcome a PR!

**Comment by aeturrell** [comment]
Closed in #78
